minio/error.py

```diff
@@ -28,8 +30,6 @@
 from xml.etree import ElementTree
 from xml.etree.ElementTree import ParseError
 
-ETREE_EXCEPTIONS = (ParseError, AttributeError, ValueError, TypeError)
```

Comment:
Did pylint throw a complaint about this ? it seems to be very helpful :(

Reply:
Yes. It says its not an exception. It was amazing to see this usage from my red hat days.